pkg/apis/config/v1alpha1/types.go

```diff
@@ -45,4 +45,15 @@ type ShootControllerConfig struct {
 	// SyncPeriod is the duration how often the controller performs its reconciliation.
 	// +optional
 	SyncPeriod *metav1.Duration `json:"syncPeriod,omitempty"`
+	// OIDCConfig is the configuration for the OIDC resources which are created for trusted shoots.
+	// +optional
+	OIDCConfig *OIDCConfig `json:"oidcConfig"`
+}
+
+// OIDCConfig is the configuration for the OIDC resources created for trusted shoots.
+type OIDCConfig struct {
+	// ClientID is the audience used in the OIDC resources created for trusted shoots.
+	// Default is "garden".
+	// +optional
+	ClientID string `json:"clientID"`
```

Comment:
I wonder if we should go ahead and call this "Audiences" and allow for multiple values (validation will restrict to a single one). This will allow eventually aligning the OIDC API with structured authentication in Kubernetes without introducing a change here. https://github.com/gardener/oidc-webhook-authenticator/blob/97e9faea574af4f1a0259ee7f5c159c1ea290b34/controllers/authentication/openidconnect_controller.go#L148